[2024-08-21T20:47:29.252+0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-21T20:47:29.273+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 3_models_dag_home.prepare_data manual__2024-08-21T15:47:16.966141+00:00 [queued]>
[2024-08-21T20:47:29.280+0500] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 3_models_dag_home.prepare_data manual__2024-08-21T15:47:16.966141+00:00 [queued]>
[2024-08-21T20:47:29.280+0500] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-21T20:47:29.293+0500] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): prepare_data> on 2024-08-21 15:47:16.966141+00:00
[2024-08-21T20:47:29.305+0500] {logging_mixin.py:190} WARNING - /home/nicolaspc/anaconda3/envs/mlops_env/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=304501) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T20:47:29.306+0500] {standard_task_runner.py:72} INFO - Started process 304534 to run task
[2024-08-21T20:47:29.306+0500] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', '3_models_dag_home', 'prepare_data', 'manual__2024-08-21T15:47:16.966141+00:00', '--job-id', '183', '--raw', '--subdir', 'DAGS_FOLDER/.ipynb_checkpoints/homework_stepik-checkpoint.py', '--cfg-path', '/tmp/tmp0zl2dgdz']
[2024-08-21T20:47:29.308+0500] {standard_task_runner.py:105} INFO - Job 183: Subtask prepare_data
[2024-08-21T20:47:29.517+0500] {task_command.py:467} INFO - Running <TaskInstance: 3_models_dag_home.prepare_data manual__2024-08-21T15:47:16.966141+00:00 [running]> on host nicolaspc-HP-Pavilion-Laptop-15-eh1xxx
[2024-08-21T20:47:29.589+0500] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nicolas Pal' AIRFLOW_CTX_DAG_ID='3_models_dag_home' AIRFLOW_CTX_TASK_ID='prepare_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-21T15:47:16.966141+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-21T15:47:16.966141+00:00'
[2024-08-21T20:47:29.590+0500] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-21T20:47:29.614+0500] {s3.py:1387} INFO - Downloading source S3 file from Bucket test-bucket-nicolas-1 with path nikolai022022/datasets/california_housing.pkl
[2024-08-21T20:47:29.627+0500] {base.py:84} INFO - Retrieving connection 'aws_default'
[2024-08-21T20:47:29.627+0500] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-08-21T20:47:41.582+0500] {homework_stepik-checkpoint.py:153} INFO - Data preparation finished.
[2024-08-21T20:47:41.585+0500] {python.py:240} INFO - Done. Returned value was: {'start_tiemstamp': '20240821 20:47', 'data_download_start': '20240821 20:47', 'data_download_end': '20240821 20:47', 'data_preparation_start': '20240821 20:47', 'data_preparation_end': '20240821 20:47'}
[2024-08-21T20:47:41.623+0500] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-21T20:47:41.624+0500] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=3_models_dag_home, task_id=prepare_data, run_id=manual__2024-08-21T15:47:16.966141+00:00, execution_date=20240821T154716, start_date=20240821T154729, end_date=20240821T154741
[2024-08-21T20:47:41.673+0500] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-21T20:47:41.698+0500] {taskinstance.py:3891} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2024-08-21T20:47:41.700+0500] {local_task_job_runner.py:240} INFO - ::endgroup::
